ndvi_forecast/3_test.ipynb

```diff
@@ -14,7 +14,7 @@
    "metadata": {},
    "source": [
     "# NDVI Forecast on Area of Interest (AOI)\n",
```

Comment:
Once you define Area of Interest as AOI, you can use AOI everywhere in the same file.

Reply:
Done